packages/commonwealth/server/routes/rules/checkAddressAgainstRule.ts

```diff
@@ -0,0 +1,41 @@
+import { success, TypedRequestBody, TypedResponse } from '../../types';
+import { DB } from '../../database';
+import RuleCache from '../../util/rules/ruleCache';
+import { AppError } from '../../util/errors';
+import checkRule from '../../util/rules/checkRule';
+
+export const CheckRuleErrors = {
+  NoRuleSpecified: 'No rule has been specified',
+  NoRuleProvided: 'No rule provided',
+  InvalidRule: 'Rule is not valid',
+  AddressNotValid: 'Address is not valid',
+};
+
+type checkAddressAgainstRuleReq = {
```

Comment:
We may want to make this a "bulk" type route, if we're going to be either checking N rules for an address, or N addresses for a rule, rather than iterating on the client side.